src/deprecated.js

```diff
@@ -287,29 +89,75 @@ pc.string.format = function (s) {
     return s;
 };
 
-pc.string.startsWith = function (s, subs) {
+string.startsWith = function (s, subs) {
     // #ifdef DEBUG
     console.warn("DEPRECATED: pc.string.startsWith is deprecated. Use String#startsWith instead.");
     // #endif
     return s.startsWith(subs);
 };
 
-pc.time = {
+export var time = {
     now: pc.now,
```

Comment:
This is still accessing pc namespace, should be something like:

```ts
import {now, Timer} from "../src/core/time";
```
```ts
export var time = {
    now: now,
    Timer: Timer
};
```

Reply:
Fixed!